rules/windows/persistence_via_update_orchestrator_service_hijack.toml

```diff
@@ -16,7 +16,61 @@ index = ["winlogbeat-*", "logs-endpoint.events.*", "logs-windows.*"]
 language = "eql"
 license = "Elastic License v2"
 name = "Persistence via Update Orchestrator Service Hijack"
-note = """## Setup
+note = """## Triage and analysis
+
+### Investigating Persistence via Update Orchestrator Service Hijack
+
+Windows Update Orchestrator Service is a DCOM service used by other components to install windows updates that are
+already downloaded. USO was vulnerable to Elevation of Privileges (any user to local system) due to an improper
```

Comment:
```suggestion
already downloaded. UsoSvc was vulnerable to Elevation of Privileges (any user to local system) due to an improper
```
When I googled windows update orchestrator service, the main acronym I found was "UsoSvc"